strings_lab.js

```diff
@@ -0,0 +1,178 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+
+let firstChar = "macbook air"
+console.log(firstChar[0])
+//
+// 2. `console.log` the length of a string.
+console.log(firstChar.length)
+
+// 3. `console.log` the last character of any string.
+
+console.log(firstChar[firstChar.length-1])
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+let myName = "voniel"
+let lastLetter = myName[myName.length-1]
+//console.log(lastLetter)
+  upperLast = lastLetter.toUpperCase()
+let split = myName.slice(0,5)
+    console.log(split + upperLast)
+
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+let variableAmount = 1000000;
+
+if (variableAmount === 1000000){
+  console.log( variableAmount + " dollars " + "(pinky)")
+} else {
+  console.log(variableAmount + " dollars")
+}
+
+// 6. Create a `verbing` code block. It should take a single string variable.
+//If its length is at least 3, it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+let verbing ="jump";
+if (verbing.endsWith("ing")){
+        let changeWord = verbing.replace("ing","ly")
+        console.log(changeWord)
+} else if (verbing.length >= 3){
+          console.log(verbing + "ing")
+}
+
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+
+let string = "itsy bitsy spider";
+
+let newString = string.slice(0,10)
```

Comment:
Would this code be able to work if I change string to "JRJ Rules And What Not"? If a code is written for only specific condition then it is called hard coding. Try to avoid that as much as possible as it is very bad practice. How can we fix this issue?